toqito/nonlocal_games/nonlocal_game.py

```diff
@@ -120,6 +121,28 @@ def from_bcs_game(cls, constraints: list[np.ndarray], reps: int = 1) -> "Nonloca
 
         return cls(prob_mat, pred_mat, reps)
 
+    def process_iteration(i, num_bob_outputs, num_bob_inputs, pred_mat_copy, num_alice_outputs, num_alice_inputs):
```

Comment:
Could we include type-hints for the function arguments and return type?

Reply:
added. done!